src/main/java/org/dcsa/ebl/model/base/AbstractShipmentLocation.java

```diff
@@ -14,6 +16,7 @@
 public class AbstractShipmentLocation extends AuditBase {
 
     @Column("location_id")
+    @JsonIgnore
     private UUID locationID;
```

Comment:
Strictly speaking, it does not belong on the TO variant and should ideally be moved to the DAO subclass.